lms/static/js/student_account/views/InstitutionLoginView.js

```diff
@@ -1,25 +1,32 @@
 ;(function (define) {
     'use strict';
-    define(['jquery', 'underscore', 'backbone'],
-        function($, _, Backbone) {
+    define([
+            'jquery',
+            'underscore',
+            'backbone',
+            'edx-ui-toolkit/js/utils/html-utils'
+        ],
+        function($, _, Backbone, HtmlUtils) {
 
         return Backbone.View.extend({
             el: '#institution_login-form',
 
             initialize: function( data ) {
-                var tpl = data.mode == "register" ? '#institution_register-tpl' : '#institution_login-tpl';
+                var tpl = data.mode === 'register' ? '#institution_register-tpl' : '#institution_login-tpl';
                 this.tpl = $(tpl).html();
                 this.providers = data.thirdPartyAuth.secondaryProviders || [];
                 this.platformName = data.platformName;
             },
 
             render: function() {
-                $(this.el).html(_.template(this.tpl)({
+                var output = HtmlUtils.template(this.tpl)({
```

Comment:
I would lose this variable as it is only accessed once, and it will make it harder for the linter to see that the HTML is escaped.